app/api/events.py

```diff
@@ -27,6 +27,14 @@ class Meta:
                           schema='TicketSchema',
                           many=True,
                           type_='ticket')
+    notification = Relationship(attribute='notification',
+                          self_view='v1.event_notification',
+                          self_view_kwargs={'id': '<id>'},
+                          related_view='v1.notification_detail',
+                          related_view_kwargs={'id': '<id>'},
```

Comment:
I think this should be event_id instead of just id. ID would give a wrong link I think. Please check and confirm. Also you need to add a separate url for NotificationDetail